src/main/app/controller/GetController.test.ts

```diff
@@ -19,6 +19,7 @@ describe('GetController', () => {
       extraEnglish: 'text',
       formState: req.session.userCase,
       hideBackButton: false,
+      hideNavigationButton: false,
```

Comment:
shouldn't this default to false?

Reply:
default should be true as you want to see the sign in/out button but in this specific page, its not there hence defaut is true.

if youre talking about in the actual get class, it does compare to see if the url is '/timed-out'